not-so-trivial-pursuit/client/src/components/question-render.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from "react";
+
+const RadioButton = ({answer, index, onValueChange}) => {
+ let id = `${index}-choice`;
+  return( 
+  <div>
+    <input type="radio" id={id} name="answer" value={answer} onChange={onValueChange}/>
+    <label for={id}>{answer}</label>
+  </div>  
+)};
+
+const QuestionRender = ({question}) => {
+  const[isRight, setIsRight] = useState(false);
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    console.log(evt.target);
+}
+  const onValueChange = (evt) => {
+    console.log(evt.target.value);
+    console.log(evt.target.value===question.correct_answer);
+    if(evt.target.value===question.correct_answer){
+      setIsRight(true);
+    }
+  }
+  return (
+    question && (
+      <div className={"question-section"}>
+      <div className="question-text">{question.question}</div>
+      <div className="answer-section">
+        <form onSubmit={handleSubmit}>
+          <RadioButton answer={question.incorrect_answers[0]} onValueChange={onValueChange}/>
```

Comment:
In the future, you could make RadioButton its own component!

Reply:
Oh the spread operator to incorporate the whole array of incorrect answers, I wouldn't have thought of that. Thank you!